test/e2e/integreatly_test.go

```diff
@@ -478,13 +475,13 @@ func checkOperandVersions(t *testing.T, f *framework.Framework, namespace string
 
 	err := f.Client.Get(goctx.TODO(), types.NamespacedName{Name: installationName, Namespace: namespace}, installation)
 	if err != nil {
-		return errors.Wrap(err, "Error getting installation CR from cluster when checking operand versions")
+		return fmt.Errorf("Error getting installation CR from cluster when checking operand versions: %w", err)
 	}
 
 	for product, version := range operandVersions {
 		clusterVersion := installation.Status.Stages[stage].Products[operator.ProductName(product)].Version
 		if clusterVersion != operator.ProductVersion(version) {
-			return errors.Wrap(err, fmt.Sprintf("Error with version of %s deployed on cluster. Expected %s. Got %s", product, version, clusterVersion))
+			return fmt.Errorf("Error with version of %s deployed on cluster. Expected %s. Got %s. %w", product, version, clusterVersion, err)
```

Comment:
@grdryn This seems to be the same case as other comments - err is irrelevant here.